CMakeLists.txt

```diff
@@ -7,9 +7,9 @@ cmake_minimum_required(VERSION 3.15)
 file(STRINGS "VERSION" pVersion)
 
 project(
-  w3emc
+  w3
```

Comment:
It's not clear we should change the name from w3emc to w3. This will require changes in build systems that otherwise would not be required...